src/main/java/Task.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Arrays;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public void setIsDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    public String toString() {
```

Comment:
I noticed you use the optional `@Override` for every single overridden function except for this one. Should there be one here too for consistency?